ngraph/core/include/ngraph/node.hpp

```diff
@@ -100,6 +100,7 @@ namespace ngraph
     NGRAPH_API
     NodeVector as_node_vector(const OutputVector& values);
     /// Returns a ResultVector referencing values.
+    NGRAPH_API
```

Comment:
@ilyachur Is it ok to expose `as_result_vector`? (it is used in onnx importer)

Reply:
I am ok this this change, if it is required for external components.